MuonAnalysis/MuonAssociators/interface/PropagateToMuonInterface.h

```diff
@@ -0,0 +1,69 @@
+#ifndef MuonAnalysis_MuonAssociators_interface_PropagateToMuonInterface_h
+#define MuonAnalysis_MuonAssociators_interface_PropagateToMuonInterface_h
+//
+//
+
+/**
+  \class    PropagateToMuonInterface PropagateToMuonInterface.h
+  "HLTriggerOffline/Muon/interface/PropagateToMuonInterface.h" \brief    Propagate an
+  object (usually a track) to the second muon station. Support for other muon
+  stations will be added on request.
+
+  \author   Giovanni Petrucciani
```

Comment:
I would either remove the name(s) altogether, or add your name too to signal that the class has been reworked.

Reply:
Removed